docs/recipes/make-stream-from-buffer.md

```diff
@@ -79,13 +79,14 @@ gulp.task('write-versions', function() {
   availableVersions.forEach(function(v) {
     // make a new stream with fake file name
     var stream = source('final.' + v);
+    
+    var streamEnd = stream;
```

Comment:
@tepez :+1: I think this is better than the original version, although `var streamEnd; ... streamEnd = stream ...` would've made sense to me also.